### ISSUE TITLE
Row order

### ISSUE THREAD

**Issue by vtrbtf** [satyrius/gonx#35]
Is there any way to get the row line ? 

I was playing around, however I really need the row number, and currently thats not possible since the reader is no sync.

**Comment by satyrius** [comment]
No, there is no line number added to a record. Sounds reasonable. You could make a PR with such functionality
